server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/model/message/view/MessageFullView.java

```diff
@@ -87,8 +87,10 @@ public Builder attachedMessages(Map<BlobId, SubMessage> attachedMessages) {
         public MessageFullView build() {
             ImmutableList<Attachment> attachments = this.attachments.build();
             ImmutableMap<BlobId, SubMessage> attachedMessages = this.attachedMessages.build();
+            boolean hasAttachment = MessageFullView.hasAttachment(attachments);
+            hasAttachment(hasAttachment);
```

Comment:
`hasAttachment(hasAttachment);` -> what does that mean? WHat does that do?

Reply:
see the correct version at later commit!